articles/TOC.yml

```diff
@@ -1,13 +1,14 @@
 - name: Welcome
   href: index.md
-- name: Getting Started
-  topicHref: install-guide/index.md
-  href: install-guide/toc.yml
 - name: Quantum computing concepts
   topicHref: concepts/index.md
   href: concepts/toc.yml
-- name: Quickstart - your first quantum program
-  href: quickstart.md
+- name: Quickstarts
```

Comment:
Are we committed to the section title "Quickstarts" if we're tucking the installation page inside? If I were looking for installation instructions, I'd certainly end up there by process of elimination, but "Quickstarts" does sound more like a section of strictly tutorial-esque things. With installation in the section I'd lean toward the old "Getting Started" sidebar title.

Reply:
Quickstarts is part of the current docs standard content model, so we're committed to it. It will be linked to from the home page and as a pre-requisite from the tutorials. It should be easily found when someone googles or bings "Install Microsoft Quantum" or "Install QDK" or "Install the Quantum Development Kit"